docs/content/develop/properties.md

```diff
@@ -0,0 +1,352 @@
+
+**Before connectedCallback**
+```html
+<pfe-foo foo="hello">
+    <h1></h1>
+</pfe-foo>
+```
+
+**After connectedCallback**
+```html
+<pfe-foo pfe-c-foo="hello">
+    <h1 pfe-c-foo="hello"></h1>
+</pfe-foo>
+```
+
+Attribute values are only copied from parent to child, never the other way around.
+
+## prefix
+
+PatternFly Elements has a convention of prefixing a component's own attribute names with `pfe-c-`, and the `prefix` field controls that.  Attributes are prefixed by default, but you may set `prefix: false` if you wish your attribute to have no prefix.
+
+This can be useful when deprecating old, non-prefixed attributes.  It's also beneficial when using built-in aria attributes to manage component states, such as `aria-hidden`:
+
+```javascript
+static get properties() {
+  return {
+    ariaHidden: {
+      type: String,
+      default: "true",
+      prefix: false
+    }
+  };
+}
+```
+
+The property name `ariaHidden` results in an attribute named `aria-hidden`.  If prefix were not set to false, the generated attribute name would be `pfe-c-aria-hidden` instead.
+
+Note: [ARIA attributes](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) must be defined as Strings, for even though attributes like `aria-hidden` seem like Booleans, they are not true boolean attributes (`aria-hidden` vs. `aria-hidden="true"`) and need to receive string values of `"true"` and `"false"`.
+
+## alias
+
+Default: `undefined`
+
+The `alias` field can be used to link two properties to each other.  `alias` expects to be given the name of another property, to which it will copy its value.  When a property value is changed, if it has an alias, the new value will be sent to the alias.
+
+Aliasing is intended to help when migrating to a new property/attribute name.  It allows for two different properties to be simultaneously "active".
+
+Aliased values do not need to have a new set of definitions for cascade or observer as those states will be managed by the attribute to which it is being aliased (see example below).
+
+Example, an `old` property whose values will be forwarded along to a `new` property.
+
+```javascript
+static get properties() {
+  return {
+    new: {
+      type: Number,
+      default: 0,
+    },
+    old: {
+      type: Number,
+      alias: "new"
+    }
+  };
+}
+```
+
+With these property definitions, changes to `old` will be "forwarded" along to `new`.
+
+```javascript
+el.source = 4;
+el.destination === 4; // true
+```
+
+Two-way aliases are allowed.
+
+```javascript
+static get properties() {
+  return {
+    a: {
+      alias: "b"
+    },
+    b: {
+      alias: "a"
+    }
+  };
+}
+```
+
+## attr
+
+Default: `undefined`
+
+The `attr` field overrides the property's default generated attribute name and specifies a custom name to be used as this property's bound attribute.
+
+```javascript
+static get properties() {
+  return {
+    foo: {
+      attr: "the-one-and-only-foo"
```

Comment:
it could totally just be me but this feels unclear as to which thing is the new source of truth, or the new attribute, is it `foo` or `the-one-and-only-foo`

Reply:
I also don't know when I would use this approach vs. setting an alias

Reply:
For the alias question, you set an alias when you want to copy a value from one prop/attr to another.  It has nothing to do with attribute names.

Reply:
Not sure if this is new or I just missed it the first time but the line below this is great, maybe you can just move it above the example?

"The example below shows a property definition which results in a property named `foo` and an attribute named `the-one-and-only-foo`.